lib/resources/dstu2/allergy_intolerance.yaml

```diff
@@ -109,6 +110,10 @@ fields:
                   {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
                    "code": "4125",
                    "display": "Ethiodized oil"}]}}
+  binding:
+    display: RxNorm
+    terminology: http://www.nlm.nih.gov/research/umls/rxnorm
```

Comment:
I believe we allow for snomed as well when the substance isn't a medication, and for negation.